config.toml.example

```diff
@@ -1,31 +1,30 @@
-# * for all
-group_search = "*"
-
-# After too long a session can expire.
-# So we just trigger a re-atuh, every reauth_every account imports.
-reauth_every = 150
-
-# We batch our group membership queries.
-# How many members maximum should be in each request?
-group_chunks = 30
 
-# Which groups should we ignore when creating and mapping?
-ignore_groups = ["cla_fpca", "cla_done", "cla_fedora"]
+# We batch our queries (groups, users, memberships, etc).
+# How many objects maximum should be in each request?
+chunks = 30
 
 # Record and replay requests to FAS (for testing)
 replay = false
 
+[groups]
+# * for all
+search = "*"
+# Which groups should we ignore when creating and mapping?
+ignore = ["cla_fpca", "cla_done", "cla_fedora"]
 
 [fas]
 url = "https://admin.fedoraproject.org/accounts"
-# username = 
-# password = 
+# username =
+# password =
 
 [ipa]
 instances = ["ipa.fas2ipa.test"]
 cert_path = "/etc/ipa/ca.crt"
 username = "admin"
 password = "adminPassw0rd!"
+# After too long a session can expire.
+# So we just trigger a re-atuh, every reauth_every imports.
```

Comment:
small typo in the comment here. s/atuh/auth/

Reply:
Woah good catch! :-)